content/docs/integrations/kubernetes/index.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Adding a Kubernetes cluster
+lastUpdated: '2021-07-15T21:00:00.0Z'
+description: How to add a Kubernetes cluster for the Kubernetes plugin.
+---
+
+![Full active cluster on roadie](./active.png)
+
+> ⚠️ Note, the Kubernetes plugin is only supported with AWS clusters. ⚠️
+> GKE support is coming soon.
+
+# AWS
+
+## Introduction
+
+In order to use the Kubernetes plugin, Roadie needs:
+ * An Assumed Role to fetch the resources from your cluster
+ * The name of your cluster
+ * URL of the loadbalancer/entry point to your Kubernetes cluster
+
+
+These are set within Roadie at the following url:
+
+```text
+https://<tenant-name>.roadie.so/administration/settings/kubernetes
+```
+
+This page describes how to create and set up the API token.
+
+## Steps
+
+In this section we will create an AWS role that will allow the roadie user to access (read only) your Kubernetes cluster's resources.
+ * [Assuming Role](https://docs.aws.amazon.com/STS/latest/APIReference/API_AssumeRole.html)
+ * [Cross Federation](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_common-scenarios_third-party.html)
+ * [Kubernetes RBAC](https://kubernetes.io/docs/reference/access-authn-authz/rbac/)
+ * [Trust relationship](https://aws.amazon.com/en/blogs/security/how-to-use-trust-policies-with-iam-roles/)
+
+### Step 1: Creating the cross account federation role
+
+1. Sign into your AWS console and navigate to the [IAM service](https://console.aws.amazon.com/iam/home#/home).
+
+2. Click on ”Role” link (this should be on the left handside of your screen).
+
+3. Click on the ”Create Role” button.
+
+4. Click on ”Another AWS Account” and add the account number on on the Kubernetes configuration page (in Roadie) and then click on ”Next: permissions”.
+
+![Another AWS Account](./role-creation.png)
+
+5. Ignore attached policies and click on ”Next: tags” (Roadie does not need to read your AWS resources, only access to your Kubernetes cluster).
```

Comment:
Have you tested this with empty role that it works?

Reply:
Yep it worked for me!

Reply:
If you look at the kubeadministrator, it has no policies. Only trust relationships.